optional/cloudfront_cdn/main.tf

```diff
@@ -22,26 +22,16 @@ resource "aws_cloudfront_distribution" "cloudfront_distribution" {
   }
 
   default_cache_behavior {
-    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cache_policy_id = aws_cloudfront_cache_policy.default_cache_policy.id
+
+    allowed_methods  = ["GET", "HEAD"]
     cached_methods   = ["GET", "HEAD"]
     target_origin_id = var.origin_domain
 
     default_ttl            = var.cache_expiration
     max_ttl                = var.cache_expiration
     compress               = true
-    viewer_protocol_policy = "allow-all"
-
-    forwarded_values {
-      query_string = true
-
-      headers = [
-        "*",
-      ]
-
-      cookies {
-        forward           = "none"
-      }
-    }
```

Comment:
@Cup0fCoffee, hmmm, after looking at this again, I can see that adding the cache policy is overkill. We are going to achieve the same result by changing `"*"` in `headers` to `"Origin"`, and changing the default value of `cache_expiration` to the max, i.e. `31536000`.

It will be simpler because the Terraform module requires [default_cache_behavior](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_distribution#default_cache_behavior) defined anyway, despite it's not being used in this scenario. Duplicating default behavior and policies will make our Terraform code less readable.

@giovannicimolin, any thoughts?

Reply:
@Agrendalath :+1: for this solution, I prefer keeping a simple approach wherever possible.

Do you think applying this will affect the current distribution for KKUx?

Reply:
@giovannicimolin, yep, a change will be required, as it changes the cache behavior in this distribution. Nothing big, though.